tshirts.cpp

```diff
@@ -16,6 +16,7 @@ char size(int cms) {
 int main() {
     assert(size(37) == 'S');

```

Comment:
Test cases need to be strengthened here to cover all boundary conditions and values before and after.